google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -988,19 +988,32 @@ public Acl blobToPublicRead(String bucketName, String blobName, long blobGenerat
   }
 
   /** Example of default auth */
-  public Page<Bucket> authListBuckets() {
+  public void authListBuckets() {
     // [START auth_cloud_implicit]
     // If you don't specify credentials when constructing the client, the
     // client library will look for credentials in the environment.
```

Comment:
will look --> looks (tech writing lives in the eternal present)

Reply:
authListBuckets() is used on https://cloud.google.com/docs/authentication/getting-started#auth-cloud-implicit-java with specific comments for that page. However it is duped on another sample in java-docs-samples for a related page: https://cloud.google.com/docs/authentication/production. Removing sample from this doc and updating documentation to point to the other sample.

listBuckets() will be used on https://cloud.google.com/storage/docs/listing-buckets

Reply:
Documentation was updated to not depend on the authListBuckets sample.